src/obr/core/core.py

```diff
@@ -142,6 +143,34 @@ def logged_func(func, doc, **kwargs):
     doc["history"].append(res)
 
 
+def merge_job_documents(job: Job):
+    """Merge multiple job_document_hash.json files into job_document.json"""
+    root, _, files = next(os.walk(job.path))
+    print(files)
+
+    def is_job_sub_document(fn):
+        if fn == "signac_job_document.json":
+            return False
+        return fn.startswith("signac_job_document")
+
+    files = [f for f in files if is_job_sub_document(f)]
+    merged_data = []
+    merged_history = []
+    cache = None
+    for f in files:
+        with open(Path(root) / f, "r") as fh:
```

Comment:
Not to nitpick, but "r" is the default mode. Consider removing it for brevity's sake
```suggestion
        with open(Path(root) / f) as fh:
```